src/Data/Biapplicative.hs

```diff
@@ -454,3 +464,24 @@ instance Biapplicative Const where
   {-# INLINE bipure #-}
   Const f <<*>> Const x = Const (f x)
   {-# INLINE (<<*>>) #-}
+
+-- | An analogue of @"Control.Applicative.Backwards".'Control.Applicative.Backwards.Backwards'@
+-- for bifunctors.
+newtype Backwards p a b = Backwards { forwards :: p a b }
+  deriving (Show, Read, Eq, Ord, Functor, Base.Bifunctor)
+
+instance Biapplicative p => Biapplicative (Backwards p) where
+  bipure :: forall a b. a -> b -> Backwards p a b
+  bipure = coerce (bipure @p @a @b)
+
+  biliftA2 f g (Backwards m) (Backwards n) = Backwards $ biliftA2 (flip f) (flip g) n m
+
+instance Applicative (p a) => Applicative (Backwards p a) where
+  pure :: forall b. b -> Backwards p a b
+  pure = coerce (pure @(p a) @b)
+
+  liftA2 f (Backwards m) (Backwards n) = Backwards $ liftA2 (flip f) n m
+
+deriving instance Bifoldable p => Bifoldable (Backwards p)
```

Comment:
This `Backwards` type has both `Functor` as `Bifunctor` instances. By analogy, should there also be `Foldable`/`Traversable` instances to accompany the `Bifoldable`/`Bitraversable` instances?